test/core/MakerOperations.t.sol

```diff
@@ -903,160 +903,160 @@ contract MakerOperationsTest is MangroveTest, IMaker {
-    int level0Index = mgv.local(olKey).bestTick().level0Index();
+    Field level2 = mgv.local(olKey).level2();
+    int level2Index = mgv.local(olKey).bestBin().level2Index();
     uint ofr = mkr.newOfferByVolume(1 ether, 10 ether, 100_000, 0);
     assertGt(
-      level0Index,
-      mgv.local(olKey).bestTick().level0Index(),
-      "test void if level0 does not change when second offer is created"
+      level2Index,
+      mgv.local(olKey).bestBin().level2Index(),
+      "test void if level2 does not change when second offer is created"
     );
     mkr.retractOffer(ofr);
-    assertEq(level0, mgv.local(olKey).level0(), "level0 should have been restored");
+    assertEq(level2, mgv.local(olKey).level2(), "level2 should have been restored");
   }
 
-  // function test_firstOffer_fuzz_price() public {
+  // function test_firstOffer_fuzz_ratio() public {
   //   mkr.provisionMgv(10 ether);
-  //   uint ofr = mkr.newOfferByLogPrice(300_000, 0.0001 ether, 100_000, 0);
+  //   uint ofr = mkr.newOfferByTick(300_000, 0.0001 ether, 100_000, 0);
   // }
 
   function test_update_branch_on_retract_level1() public {
     mkr.provisionMgv(10 ether);
     mkr.newOfferByVolume(1.0 ether, 1 ether, 100_000, 0);
     Field level1 = mgv.local(olKey).level1();
-    int level1Index = mgv.local(olKey).bestTick().level1Index();
+    int level1Index = mgv.local(olKey).bestBin().level1Index();
     uint ofr = mkr.newOfferByVolume(1 ether, 100 ether, 100_000, 0);
     assertGt(
       level1Index,
-      mgv.local(olKey).bestTick().level1Index(),
+      mgv.local(olKey).bestBin().level1Index(),
       "test void if level1 does not change when second offer is created"
     );
     mkr.retractOffer(ofr);
     assertEq(level1, mgv.local(olKey).level1(), "level1 should have been restored");
   }
 
-  function test_update_branch_on_retract_level2() public {
+  function test_update_branch_on_retract_level0() public {
     mkr.provisionMgv(10 ether);
     mkr.newOfferByVolume(1.0 ether, 1 ether, 100_000, 0);
-    Field level2 = mgv.local(olKey).level2();
+    Field level0 = mgv.local(olKey).level0();
     uint ofr = mkr.newOfferByVolume(1 ether, 100 ether, 100_000, 0);
     assertTrue(
-      !level2.eq(mgv.local(olKey).level2()), "test void if level2 does not change when second offer is created"
+      !level0.eq(mgv.local(olKey).level0()), "test void if level0 does not change when second offer is created"
     );
     mkr.retractOffer(ofr);
-    assertEq(level2, mgv.local(olKey).level2(), "level2 should have been restored");
+    assertEq(level0, mgv.local(olKey).level0(), "level0 should have been restored");
   }
 
   function test_update_branch_on_insert_posInLeaf() public {
     mkr.provisionMgv(10 ether);
-    Tick tick0 = Tick.wrap(0);
-    mkr.newOfferByLogPrice(LogPriceLib.fromTick(tick0, olKey.tickScale), 1 ether, 100_000, 0);
-    uint ofr = mkr.newOfferByLogPrice(-46055, 100 ether, 100_000, 0);
+    Bin tick0 = Bin.wrap(0);
```

Comment:
```suggestion
    Bin bin0 = Bin.wrap(0);
```

Reply:
fixed this and related ones in ab9762cf2474ac16ec0ec0b370ab3a53141e7dbd